packages/modelviewer.dev/data/docs.json

```diff
@@ -5,65 +5,65 @@
       {
         "name": "src",
         "htmlName": "src",
-        "description": "<p>The URL to the 3D model. This parameter is required for <span class=\"attribute\">&lt;model-viewer&gt;</span> to display. Only <a href=\"https://github.com/KhronosGroup/glTF/tree/master/specification/2.0\">glTF</a>/<a href=\"https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#glb-file-format-specification\">GLB</a> models are supported, see <a href=\"https://github.com/google/model-viewer#supported-formats\">Supported Formats</a>.</p>",
+        "description": "The URL to the 3D model. This parameter is required for <span class=\"attribute\">&lt;model-viewer&gt;</span> to display. Only <a href=\"https://github.com/KhronosGroup/glTF/tree/master/specification/2.0\">glTF</a>/<a href=\"https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#glb-file-format-specification\">GLB</a> models are supported, see <a href=\"https://github.com/google/model-viewer#supported-formats\">Supported Formats</a>.",
         "default": [
           "String",
-          "any legal filepath",
-          "empty string"
+          "empty string",
```

Comment:
I should have noticed this earlier; these should probably be "type", "default", and "options" instead of an array. Might be easiest to write a short script to do that conversion.

Reply:
done